internal/rbd/rbd_util.go

```diff
@@ -265,11 +265,16 @@ func HexStringToInteger(hexString string) (uint, error) {
 // isKrbdFeatureSupported checks if a given Image Feature is supported by krbd
 // driver or not.
 func isKrbdFeatureSupported(ctx context.Context, imageFeatures string) bool {
+	supported := true
+	// return true when /sys/bus/rbd/supported_features is absent
+	// ex: kernel < v4.11.0
+	if _, err := os.Stat(krbdSupportedFeaturesFile); errors.Is(err, os.ErrNotExist) {
+		return supported
+	}
```

Comment:
if `krbdFeatures` is 0 means we are not able to get the `supported_featurs`  we don't need to do stat again and again?